Assets/HoloToolkit/Utilities/Scripts/CameraCache.cs

```diff
@@ -0,0 +1,30 @@
+﻿using UnityEngine;
+
+namespace HoloToolkit.Unity
```

Comment:
Should the CameraCache be in the HoloToolkit.Unity.Input namespace?  That is where our other camera related stuff lives right now.